.gitignore

```diff
@@ -34,3 +34,6 @@ coverage/public/packs
 yarn-debug.log*
 .yarn-integrity
 package-lock.json
+
+
+*.pdf
```

Comment:
is this pdf related to raiting?

Reply:
No, I had a lot of pdf generated in my root folder  and I wanted to ignore them